### ISSUE TITLE
Unable to remove 'WIP' from PR title

### ISSUE THREAD

**Issue by chr15m** [go-gitea/gitea#30363]
### Description

To replicate:

- Create a PR with the name 'WIP: Testing'
- Try to change the title to 'Testing'
- Observe errors on the console

```
POST https://REDACTED.com/REDACTED/REDACTED/pulls/488/t_repo-issue.js:554
Status: 500 (Internal Server Error)
```

```
JavaScript error: Cannot read properties of undefined (reading 'endsWith') (https://b1fed6fb.hostedgitea.com/assets/js/index.js?v=1.21.9 @ 20:181341). Open browser console to see more details.
```

```
Uncaught (in promise)
{
  readyState: 4,
  getResponseHeader: f,
  getAllResponseHeaders: f,
  setRequestHeader: f,
  overrideMimeType: f,
  ...
}
```

### Gitea Version

1.21.9

### Can you reproduce the bug on the Gitea demo site?

No

### Git Version

2.17.1

### Operating System

Ubuntu 18

### How are you running Gitea?

Downloaded binary release.

### Database

SQLite

**Comment by February30th** [comment]
Already fixed on v1.21.10

**Comment by chr15m** [comment]
Oh, my apologies! Thank you.
